### ISSUE TITLE
Invalid input, no valid skylines_key given

### ISSUE THREAD

**Issue by ubx** [kerel-fs/ogn-skylines-gateway#5]
vagrant@debian-jessie:/vagrant$ ./manage.py users.insert "D4414AC"  "4B5171"
Invalid input, no valid skylines_key given

Both, ogn and Skyline live tracking keys are valid

**Comment by kerel-fs** [comment]
tracking_keys are generated by

```
tracking_key = struct.unpack('I', os.urandom(4))[0]
```

and presented as

```
'{:X}'.format(tracking_key)
```

[source](https://github.com/skylines-project/skylines/blob/master/skylines/model/user.py#L212)

**Comment by kerel-fs** [comment]
Thanks @ubx!  Fixed the validation with commit ad60e84.
